roles/openshift_sanitize_inventory/tasks/__deprecations_logging.yml

```diff
@@ -21,16 +21,22 @@
       openshift_logging_image_pull_secret: openshift_hosted_logging_image_pull_secret
       openshift_logging_kibana_hostname: openshift_hosted_logging_hostname
       openshift_logging_kibana_ops_hostname: openshift_hosted_logging_ops_hostname
+      openshift_logging_kibana_nodeselector: openshift_hosted_logging_kibana_nodeselector
+      openshift_logging_kibana_ops_nodeselector: openshift_hosted_logging_kibana_ops_nodeselector
       openshift_logging_fluentd_journal_source: openshift_hosted_logging_journal_source
       openshift_logging_fluentd_journal_read_from_head: openshift_hosted_logging_journal_read_from_head
+      openshift_logging_fluentd_nodeselector: openshift_hosted_logging_fluentd_nodeselector_label
       openshift_logging_es_memory_limit: openshift_hosted_logging_elasticsearch_instance_ram
       openshift_logging_es_nodeselector: openshift_hosted_logging_elasticsearch_nodeselector
+      openshift_logging_es_ops_nodeselector: openshift_hosted_logging_elasticsearch_ops_nodeselector
       openshift_logging_es_ops_memory_limit: openshift_hosted_logging_elasticsearch_ops_instance_ram
       openshift_logging_storage_access_modes: openshift_hosted_logging_storage_access_modes
       openshift_logging_master_public_url: openshift_hosted_logging_master_public_url
       openshift_logging_image_prefix: openshift_hosted_logging_deployer_prefix
       openshift_logging_image_version: openshift_hosted_logging_deployer_version
       openshift_logging_install_logging: openshift_hosted_logging_deploy
+      openshift_logging_curator_nodeselector: openshift_hosted_logging_curator_nodeselector
```

Comment:
so i just remembered why we weren't doing this before... the openshift_hosted_logging_*_nodeselectors used the form "key=value" and the new variables use "{key: value}".

TBH I'm surprised there wasn't a bz that the es node selectors didn't work anymore, should be alright in that case then. We may need to ensure docs reflect the proper format...